pimd/pim_cmd.c

```diff
@@ -1285,6 +1285,8 @@ static void igmp_show_statistics(struct pim_instance *pim, struct vty *vty,
 {
 	struct interface *ifp;
 	struct igmp_stats igmp_stats;
+	bool found_ifname = false;
+	json_object *json = json_object_new_object();
```

Comment:
This will lead to memory leak when this api gets called without json option.

Reply:
Right, will fix.

Reply:
Fixed.